concent_api/verifier/tasks.py

```diff
@@ -47,6 +48,19 @@ def blender_verification_order(
     output_format: str,
     scene_file: str,  # pylint: disable=unused-argument
 ):
+    log_string_message(
+        logger,
+        f'Blender_verification_order_starts. SUBTASK_ID: {subtask_id} ',
+        f'Source_package_path {source_package_path} ',
```

Comment:
You should add some speparator between messages. As it is now it will be hard to tell where one message ends and the other starts.

Also, consider putting the values in quotes.

Reply:
Done. Added colon in messages and dot between messages.